app/style/auth/account.less

```diff
@@ -131,4 +131,16 @@
       }
     }
   }
+
+  #wire-login-username,
```

Comment:
These fields are not auto completed - see [auth.html](https://github.com/wireapp/wire-webapp/blob/dev/app/page/auth.html#L136).
You are right, my PR actually only fixes #614, not #198.